cirrus/server/cirrus/main.py

```diff
@@ -48,6 +51,21 @@ async def lifespan(app: FastAPI):
     Glean.shutdown()
 
 
+def initialize_sentry():
+    if cirrus_sentry_dsn:  # pragma: no cover
+        sentry_sdk.init(
+            dsn=cirrus_sentry_dsn,
+            # Set traces_sample_rate to 1.0 to capture 100%
+            # of transactions for performance monitoring.
+            # We recommend adjusting this value in production.
+            traces_sample_rate=1.0,
+            # Set profiles_sample_rate to 1.0 to profile 100%
+            # of sampled transactions.
+            # We recommend adjusting this value in production.
+            profiles_sample_rate=1.0,
```

Comment:
Yeah so here we'll want to set this to something conservative like 0.1, we don't have the credit budget to send everything like we do for Experimenter.